packages/app/src/utils/constants.ts

```diff
@@ -1,5 +1,7 @@
 import { checksumAddress } from "./formatters";
 
+export const NATIVE_TOKEN_L2_ADDRESS = "0x000000000000000000000000000000000000800a";
```

Comment:
On UI we have configs for every network, this value should be there: 
https://github.com/matter-labs/block-explorer/blob/main/packages/app/src/configs/dev.config.json

It also supports hyperchain config so you can your custom Base token address configured there.

Reply:
I don't think I quite understood this message, do you mean that each network from dev.config.json must have a "baseTokenL2Address" field?

Reply:
@fkrause98 yes, instead of having the base token address hardcoded in constants I think it should be a part of the [network configuration](https://github.com/matter-labs/block-explorer/blob/main/packages/app/src/configs/index.ts#L1). This way we will be able to easily configure the base token per network.

Reply:
Done! Now the network type also has a baseTokenAddress field, which I've used to replace the l2 address constant.